src/components/Greetings.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../store/greetingsSlice';
+
+function Greetings() {
+  const dispatch = useDispatch();
+  const greetingsData = useSelector((state) => state.greetings.greetings);
+  const singleGreeting = greetingsData[Math.floor(Math.random() * greetingsData.length)];
```

Comment:
- [ ] According to the requirements, you should receive only one Greeting from your server API. Please consider refactoring this logic to meet this! 🤓 🔧